README.md

```diff
@@ -1,6 +1,93 @@
 # Look
 
-## History
+**Look: The Generic Distributed Materialized Views Engine.**
+
+_By engineers, for engineers, crowdsourced under the umbrella of the SysDesign Meetup._
+
+## Product Vision
+
+Look is the ultimate high-performance materialized view.
+
+It aggregates data mutations from various sources, applies these mutations to a single large, holistic, mostly-in-memory data blob, and exposes interfaces to run queries against this data, at its most recent snapshot as well as against its past.
```

Comment:
Good catch! I should make it more specific indeed.

It can be a CRDT, or can be some form of last-write-wins. The beauty of the solution is that's up to the author of the respective handler.

For example, when implementing Redis, "HSET" would be last-write-wins, while INCR, well, it won't matter in practice, but that's more of a CRDT.

Do you think an example would do the job, or shall I try to phrase it better in text?

Reply:
This is actually one of the key points on this text, so I'm both delighted you pointed this out and frustrated the text as it is right now does not highlight it well.

The system I'd like to see built is generic and agnostic to whether the very data processing logic should be CRDT, or last-write-wins, or whichever "throw some exception and append to some DLQ if a conflict is detected". I'll keep thinking how to best insert this idea into the text while still keeping it short and concise,